<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageLines</class>
 <widget class="QDialog" name="ManageLines">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>添加</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icon/icon/add.png</normaloff>:/icon/icon/add.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="tabAddLine">
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <attribute name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icon/icon/subway.png</normaloff>:/icon/icon/subway.png</iconset>
    </attribute>
    <attribute name="title">
     <string>线路(&amp;L)</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;线路名称：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="buddy">
      <cstring>lineEditLineName</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLineName">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入线路名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddLine">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>添加路线</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;线路颜色：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="buddy">
      <cstring>pushButtonChooseColor</cstring>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonChooseColor">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>选择颜色</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabAddStation">
    <attribute name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icon/icon/station.png</normaloff>:/icon/icon/station.png</iconset>
    </attribute>
    <attribute name="title">
     <string>站点(&amp;S)</string>
    </attribute>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxLatitude">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>85</width>
       <height>20</height>
      </size>
     </property>
     <property name="decimals">
      <number>7</number>
     </property>
     <property name="minimum">
      <double>30.899999999999999</double>
     </property>
     <property name="maximum">
      <double>31.449999999999999</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>30.899999999999999</double>
     </property>
    </widget>
    <widget class="QLabel" name="labelLatitude">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>230</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>北纬&amp;N</string>
     </property>
     <property name="buddy">
      <cstring>doubleSpinBoxLatitude</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="labelLongitude">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>200</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>东经&amp;E</string>
     </property>
     <property name="buddy">
      <cstring>doubleSpinBoxLongitude</cstring>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxLongitude">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>85</width>
       <height>20</height>
      </size>
     </property>
     <property name="decimals">
      <number>7</number>
     </property>
     <property name="minimum">
      <double>121.000000000000000</double>
     </property>
     <property name="maximum">
      <double>122.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>121.000000000000000</double>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditStationName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>添加站点名</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserLinesSelected">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>50</y>
       <width>140</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>站点名称：</string>
     </property>
     <property name="buddy">
      <cstring>lineEditStationName</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> 经纬度坐标：</string>
     </property>
     <property name="buddy">
      <cstring>doubleSpinBoxLongitude</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>已选择：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddStation">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>280</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>添加站点</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>选择 添加到线路：</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>200</width>
       <height>270</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabAddConnection">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icon/icon/connect.png</normaloff>:/icon/icon/connect.png</iconset>
    </attribute>
    <attribute name="title">
     <string>连接(&amp;C)</string>
    </attribute>
    <widget class="QComboBox" name="comboBoxConnectStation1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;站点1：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxConnectStation1</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;站点2：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxConnectStation2</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxConnectStation2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>290</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>连接</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;所属线路：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxConnectLine</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxConnectLine">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabAddByText">
    <attribute name="title">
     <string>文本方式</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>560</width>
       <height>300</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddByText">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>330</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>以文本方式添加</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
